test/http2/http2.test.js

```diff
@@ -3,6 +3,9 @@
 const semver = require('semver')
 const tap = require('tap')
 
+// TODO: check if fail the test if Node.js version does not satisfy required one in package.json ... wip
+
+// TODO: check if cleanup/update in this branch, or in the other for code changes with Node.js 8 features ... wip
```

Comment:
we can just rename the required files to `*.test.js`, there is no need for this file to exists anymore

Reply:
sorry I don't fully understand what I should do here ...

Reply:
Rename all the files inside the `if`, following this pattern `FILE.js` -> `file.test.js`, like:

`./plain.js` -> `./plain.test.js`
`./secure` -> ` ./secure.test.js` 

Then, remove `http2.test.js` completely.